packages/components/src/internal/components/settings/DeleteProjectModal.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { ChangeEventHandler, FC, memo, useCallback, useEffect, useMemo, useState } from 'react';
```

Comment:
Note that we have deliberately left the modal tall. I chatted with Keegan and we agreed that the alternative—a scrollable table with a visible scrollbar—was less desirable.